code/modules/atmospherics/machinery/other/meter.dm

```diff
@@ -55,7 +55,7 @@
 	PIPING_LAYER_DOUBLE_SHIFT(src, target_layer)
 
 /obj/machinery/meter/process_atmos()
-	if(!target)
+	if(!(target?.flags_1 & INITIALIZED_1))

```

Comment:
Here i halt meter that trying to process uninitialized pipe. It appears when we init template bounds, first it init all machinery and next pipes, but meter is machinery.